src/components/Badge/BadgeReport.jsx

```diff
@@ -303,7 +303,7 @@ const BadgeReport = props => {
                 <th style={{ width: '110px' }}>Modified</th>
                 <th style={{ width: '110px' }}>Earned Dates</th>
                 <th style={{ width: '90px' }}>Count</th>
-                {hasPermission(props.role, 'deleteOwnBadge', roles, props.permissionsUser) ? (
+                {hasPermission(props.role, 'deleteBadges', roles, props.permissionsUser) ? (
```

Comment:
The function is being calling multiple times, I think it's better if you create a variable
```suggestion
                const hasDeleteBadgedPermission = hasPermission(props.role, 'deleteOwnBadge', roles, props.permissionsUser);
                ...
                {hasDeleteBadgedPermission ? (
```